.github/workflows/ci.yml

```
@@ -17,7 +17,8 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.0
     with:
       license-check: true
-      lint: true
\ No newline at end of file
+      lint: true
+      node-versions: '["16", "18", "20"]'
```

```suggestion
      node-versions: '["18", "20", "22"]'
```
Rest of the `v5` next branches are testing using these.